app/site.rb

```diff
@@ -29,6 +29,7 @@ def route(sub='')
 		def fn
 			@options[:fn]
 		end
+		alias_method :name, :fn
```

Comment:
Resources should have an `fn` that just happens to often be name, and then we can use that for breadcrumbs